resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -201,6 +203,16 @@ public DnsNameResolverBuilder authoritativeDnsServerCache(AuthoritativeDnsServer
         return this;
     }
 
+    /**
+     * Sets the udp client listen local Address
+     * @param localAddress
+     * @return {@code this}
+     */
+    public DnsNameResolverBuilder localAddress(InetSocketAddress localAddress) {
```

Comment:
Should we make this `SocketAddress` just for the sake of be consistent with `Bootstrap` ?

Reply:
Sorry, I don't know I can commit the suggestion, I Fixed manually